### ISSUE TITLE
Messages / Search bar / search filter dropdown menu fixes

### ISSUE THREAD

**Issue by imagiMary** [status-im/status-desktop#10184]
Version: V0.11.0-rc.2
OS: Ubuntu 22.10

tldr:
Messages / Search overlay / search filter dropdown menu has the following issues:
- Community channels are not in the same order as displayed in said communities secondary nav
- Channel emoji icons don’t fit the menus channel icon circles
- The search menu contains communities that the user is not a member of

***

##Scenario 1

https://user-images.githubusercontent.com/124154641/230044432-d5e7fbd6-7a1f-48f2-83de-4d535c81ac56.mp4

&nbsp;
Preconditions: User needs to be a member of a community that has multiple channels, some of which have emoji icons. 

<ins>Steps to reproduce:</ins>
1. Navigate to messages.

2. Click in the search box at the top of the messages secondary nav bar to open the search dialog.

3. Click on the ‘In: [message/community name]’ search filter button to open the dropdown menu.

4. In the search filter dropdown menu hover over or click a community that has multiple channels, some of which have emojis.     

<ins>What currently incorrectly happens:</ins>

5. The list of channel names shown in the community search filter are:
- not in the correct order (as per the order displayed in the communities secondary nav)
- the channel emojis do not fit the channel icon circle 

<ins>What is supposed to happen:</ins>

5. The list of channel names shown in the community search filter list should:
- be in the same order as displayed in the communities secondary nav
- the emojis should fit the channel icon circle 

##Scenario 2

https://user-images.githubusercontent.com/124154641/230044488-fabb3900-50ea-444f-a0ee-6f57a70fd35a.mp4

&nbsp;

Preconditions: User has visited a community(s) that they are not a member of.

<ins>Steps to reproduce:</ins>
1. Navigate to messages.

2. Click in the search box at the top of the messages secondary nav bar to open the search dialog. 

3. Click on the ‘In: [message/community name]’ search filter button to open the dropdown menu.

<ins>What currently incorrectly happens:</ins>

4. The search filter dropdown menu shows communities that the user: is not a member but has visited, has a pending membership request with, has left the community, or has been kicked out/banned from the community - this is incorrect.

<ins>What is supposed to happen:</ins>

4. The search filter dropdown menu should only contain Communities that the user is currently a member of.

&nbsp;

**Comment by iurimatias** [comment]
moving to 2.39 due to lack of space in the milestone

**Comment by jrainville** [comment]
@noeliaSD FYI, I changed the label to your team, since it's basically only UI issues. Not a huge priority though, so feel free to move to a later milestone
